pyftpdlib/handlers.py

```diff
@@ -1773,7 +1773,7 @@ def push_dtp_data(self, data, isproducer=False, file=None, cmd=None):
         """
         if self.data_channel is not None:
             self.respond(
-                "125 Data connection already open. Transfer starting.")
+                "150 Data connection already open. Transfer starting.")
```

Comment:
https://en.wikipedia.org/wiki/List_of_FTP_server_return_codes suggests that `125` was the correct value here?

Reply:
You're totally right. This is a leftover of me playing around with the library and a buggy FTP client. I'll revert this.